src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/eventDetailsBuilder.hpp

```diff
@@ -229,6 +231,23 @@ class TEventDetailsBuilder final : public AbstractHandler<std::shared_ptr<TScanC
                 auto vulnerabilityDetection = PolicyManager::instance().getVulnerabilityDetection();
                 ecsData["wazuh"]["cluster"]["name"] = data->clusterName();
                 ecsData["wazuh"]["schema"]["version"] = WAZUH_SCHEMA_VERSION;
+                switch (data->affectedComponentType())
+                {
+                    case AffectedComponentType::Package:
+                        ecsData["wazuh"]["vulnerability"]["source"] = TGlobalData::instance().adpDescriptions().at(
+                            m_databaseFeedManager->getCNA(data->packageFormat().data(),
+                                                          data->packageSource().data(),
+                                                          data->packageVendor().data(),
+                                                          data->osPlatform().data()));
+                        break;
+
+                    case AffectedComponentType::Os:
+                        ecsData["wazuh"]["vulnerability"]["source"] =
+                            TGlobalData::instance().adpDescriptions().at("nvd");
```

Comment:
Nitpick: we can use `DEFAULT_CNA` instead of `"nvd"`

Reply:
Question: should we make the code flexible here to allow in the future the detection of OS CVEs with another CNA different than NVD?

Reply:
We can discuss this with the team, but the osScanner code is already that rigid, I just moved that logic here too. If in the future we can detect some OS vulnerabilities with another feed, we also have to change the code here https://github.com/wazuh/wazuh/blob/bde6280610f6571baad79db0e4f344cd41f76af3/src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/osScanner.hpp#L280-L281 
by adding something similar to what is here https://github.com/wazuh/wazuh/blob/bde6280610f6571baad79db0e4f344cd41f76af3/src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/packageScanner.hpp#L202

Reply:
We'll leave it like it is as we talked in the meeting, thank you.